CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## 1.2.5
+
+### Deliverables
+
+The application is primarily hosted on pypi at https://pypi.org/project/mbed-cloud-sdk and can be installed using pip:
+
+```
+$ pip install mbed-cloud-sdk
+```
+
+### Changes
+
+- Added 'last_update_time' to Groups
+- Added 'device_mode' to Certificate
+- Renamed 'owner' -> 'owner_id' on ApiKey
+- Re-implemented PaginatedResponse
+- Improvements to BaseObject data handling
```

Comment:
Is there any user facing impact of these?

- Re-implemented PaginatedResponse
- Improvements to BaseObject data handling

Reply:
PaginatedResponse is better all-round, has more methods and stuff on it. But the fundamental behaviour of being iterable is still there. It just behaves a lot like a django query object now.

BaseObject stuff should have no noticeable effect, except being a lot less code and possibly faster because it uses comprehensions.